ci/scripts/python_test.sh

```diff
@@ -52,4 +52,4 @@ export PYARROW_TEST_ORC
 export PYARROW_TEST_PARQUET
 export PYARROW_TEST_S3
 
-pytest -r s -v ${PYTEST_ARGS} --pyargs pyarrow.tests.test_orc
+pytest -r s -v ${PYTEST_ARGS} --pyargs pyarrow
```

Comment:
Wow, it's a bummer that this went through (and it might be my fault).

Reply:
It's hard to spot. We exercise the python tests with different scripts as well (e.g. when building the wheels), so we would have noticed it from the nightlies.